chart/assets/operations/instance_groups/cc-worker.yaml

```diff
@@ -9,8 +9,3 @@
         command: [/usr/bin/pgrep, --full, cc-worker-cloud_controller_worker]
   {{- end }}
 {{- end }}
-
-
-{{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/cc-worker_*" }}
```

Comment:
Hmm, don't we have a [cc-worker patch](https://github.com/cloudfoundry-incubator/kubecf/blob/7c852467748e367b1ec7349747c3e5e4f691e001/bosh/releases/pre_render_scripts/cc-worker/cloud_controller_worker/jobs/patch_cloud_controller_worker_erb.sh) still? Why do we no longer need to add it explicitly?

Reply:
Thanks for catching that; we do indeed still need this. I've amended the "revert" commit to except this change from the revert.

Reply:
Odd that we still have a single line change here, but that's good enough :)